romanisim/l3.py

```diff
@@ -87,12 +89,14 @@ def add_objects_to_l3(l3_mos, source_cat, exptimes, xpos=None, ypos=None, coords
 
     # Add sources to the original mosaic data array
     romanisim.image.add_objects_to_image(sourcecountsall, source_cat, xpos=xpos, ypos=ypos,
-                                         psf=psf, flux_to_counts_factor=[xpt * unit_factor for xpt in exptimes],
-                                         exptimes=exptimes, bandpass=[filter_name], filter_name=filter_name,
-                                         wcs=wcs, rng=rng, seed=seed)
+                                         psf=psf, flux_to_counts_factor=exptimes,
```

Comment:
flux_to_counts_factor still needs the factor of abflux?

Reply:
Should all be addressed.